### PR TITLE
Add Raw interface for network,volume inspect

### PR DESCRIPTION
**Pull request by HackToday**
Closes: #207

### CODE REVIEW

**Comment by HackToday** [comment]
ping @vdemeester and @thaJeztah  PTAL let me know if it ok or not for this. Thanks

File: client/volume_inspect.go [hunk 1]
```diff
@@ -22,3 +24,24 @@ func (cli *Client) VolumeInspect(ctx context.Context, volumeID string) (types.Vo
 	ensureReaderClosed(resp)
 	return volume, err
 }
+
+// VolumeInspectWithRaw returns the information about a specific volume in the docker host and it's raw representation
+func (cli *Client) VolumeInspectWithRaw(ctx context.Context, volumeID string) (types.Volume, []byte, error) {
```
Comment (dnephin) [inline client/volume_inspect.go]: To remove the code duplicate with `VolumeInspect()` maybe `VolumeInspect()` should call this new method, and just discard the body?

This method does a little bit of extra work, but it shouldn't be significant, and the old method is really just there for backwards compatibility now (it won't actually get called by the engine I believe).

Same comment about the network inspect call.

Comment (vdemeester) [inline client/volume_inspect.go]: @HackToday I definitely agree with @dnephin. I even think we could do this for the existing ones (in `container_inspect.go` and `image_inspect.go`).

**Comment by HackToday** [comment]
@dnephin  and @vdemeester  PTAL the new change. Thanks

**Comment by vdemeester** [comment]
LGTM 🐮

**Comment by calavera** [comment]
LGTM

**Pull request merged**